python/py-flake8/Portfile

```diff
@@ -28,7 +28,7 @@ checksums           rmd160  3d26d25023445ef6462dabb50503736d19c3c597 \
                     sha256  c61007e76655af75e6785a931f452915b371dc48f56efd765247c8fe68f2b181 \
                     size    138524
 
-python.versions     37 38 39 310 311
+python.versions     37 38 39 310 311 312
```

Comment:
you have to add the select file as well ; that is the case for some of the other ports as well. Please verify.

Reply:
@reneeotten can we start from https://github.com/macports/macports-ports/pull/21733?

the only reason why I’ve spilt it into servers PR (and a few is in the queue): GitHub has its limit 30 or a bit less commits

Reply:
BTW I've tacked your comment into account and this is fixed and all affected ports are verified.

Reply:
I merged the initial PR - please rebase this one.

Reply:
Here it is. Rebase also included verify that files for select group are added.